css/properties/scroll-timeline-axis.json

```
@@ -7,6 +7,10 @@
           "spec_url": "https://drafts.csswg.org/scroll-animations/#propdef-scroll-timeline-axis",
           "support": {
             "chrome": [
+              {
+                "version_added": "115",
+                "notes": "Supports the <code>x</code> and <code>y</code> values, and not the deprecated <code>horizontal</code> and <code>vertical</code> values."
```

I'd leave this out. Only have a note when things are not according to the standard. Like in the case of Firefox.

```suggestion
                "version_added": "115"
```